akka-persistence-typed/src/main/scala/akka/persistence/typed/internal/PersistentActorImpl.scala

```diff
@@ -40,9 +33,8 @@ import akka.persistence.journal.Tagged
  * The `PersistentActor` that runs a `PersistentBehavior`.
  */
 @InternalApi private[akka] class PersistentActorImpl[C, E, S](
-  behavior: PersistentBehavior[C, E, S]) extends UntypedPersistentActor {
+  behavior: PersistentBehavior[C, E, S]) extends UntypedPersistentActor with ActorLogging {
```

Comment:
all this impl will go to hell with the reimpl as native typed behavior 😈  (half joking, I'll use parts of this)